src/components/UserPage/UserPage.tsx

```diff
@@ -130,6 +133,9 @@ export const UserPage = ({ userId }: UserPageProps) => {
                     {nullable(user.meta.isBonusViewable, () => (
                         <UserBonusPoints user={user} />
                     ))}
+                    {nullable(user.achievements.length || data?.access.achievement, () => (
```

Comment:
I think we don't need this nullable, just show achievement list for everybody. Later we will hide them using user settings

Reply:
but if where user.achievements.length === 0, why should we show achievements?

Reply:
This check can be inside achievement component. Also, how would you add new achievement if the ui is hidden?

Reply:
yeah, you are wright